app/proc/telegram.go

```diff
@@ -48,18 +51,97 @@ func (client TelegramClient) Send(channelID string, item feed.Item) error {
 		return nil
 	}
 
+	contentLength, err := getContentLength(item.Enclosure.URL)
+	if err != nil {
+		return err
+	}
+
+	var message *tb.Message
+
+	if contentLength < 50_000_000 {
```

Comment:
moving 50_000_000 to a const and naming it logically, like `maxTelegramFileSize` or smth like this will be nice